dotCMS/src/main/java/com/dotmarketing/util/Config.java

```diff
@@ -473,31 +415,44 @@ public static String getStringProperty (String name) {
 		return entities;
 	}
 
-	/**
-	 * If config value == null, returns the default
-	 * @param name
-	 * @param defaultValue
-	 * @return
-	 */
-  public static String[] getStringArrayProperty (String name, String[] defaultValue) {
-      _refreshProperties ();
-      
-      return props.containsKey(name) ? props.getStringArray(name) : defaultValue;
-  }
+    /**
+     * If config value == null, returns the default
+     * 
+     * @param name
+     * @param defaultValue
+     * @return
+     */
+    public static String[] getStringArrayProperty(final String name, final String[] defaultValue) {
```

Comment:
Codacy found an issue: [Consider using varargs for methods or constructors which take an array the last parameter.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5943250)